utils.h

```diff
@@ -176,7 +175,12 @@ static inline int is_cue_url(const char *name)
 
 static inline int is_url(const char *name)
 {
-	return is_http_url(name) || is_cdda_url(name) || is_cue_url(name);
+	return is_http_url(name) || is_cdda_url(name);
+}
+
+static inline int is_partial_url(const char *name)
```

Comment:
I can revert it back to the previous state. It's impossible to kick out `is_cue_url` and `is_cdda_url` in the current state of codebase. Filetype detection (made using awkward enum) and other specific properties hardly depend on them.